tests/test_interface_private_vlan.rb

```diff
@@ -61,7 +61,8 @@ class TestSwitchport < TestInterfaceSwitchport
   def test_interface_switchport_private_host_mode
     if validate_property_excluded?('interface',
                                    'switchport_mode_private_vlan_host')
-      assert_nil(interface.switchport_mode_private_vlan_host)
+      assert_equal([], interface.switchport_mode_private_vlan_host,
+                   'Err: should not be supported ')
```

Comment:
These asserts don't make sense. The getters are going to return [] even if the properties are supported. 
Just remove them altogether -- the assert_raises should be sufficient for these properties.